### ISSUE TITLE
binary:logitraw

### ISSUE THREAD

**Issue by lwm345** [dmlc/xgboost#2275]
what is the differents between reg:linear and binary:logitraw in xgboost?

**Comment by lwm345** [comment]
what is the differents between reg:linear and binary:logitraw ?

**Comment by pommedeterresautee** [comment]
It's not the same loss being optimized 
https://github.com/dmlc/xgboost/blob/master/doc/parameter.md#learning-task-parameters
